xml/deployment_installation.xml

```diff
@@ -715,26 +741,352 @@
+      Configuration chapter of this manual; for more information, see
+      <xref linkend="cloud-init.configuration"/>.
+     </para>
+     <para>
+      As with individual node installation, the &admin_node; must be configured
+      separately and differently from the &worker_node;s.
+     </para>
+    </sect4>
+    <sect4>
+     <title>Procedure</title>
+     <para>
+      The process is very similar for all hypervisors. The examples in this
+      manual use &kvm; running on &sle;.
+     </para>
+     <para>      
+      You must build at least two ISO images with configuration information:
+      one for the &admin_node; and one for the &worker_node;s. If you wish to
+      assign specific host names or customize individual &worker_node;s, then
+      you must create separate ISO images for each node containing the
+      customised files.
+     </para>
+     <para>
+      Additionally, you must make a separate copy of the downloaded VM disk
+      image for each VM. We suggest keeping the original download elsewhere and
+      making a fresh copy for each node, naming it appropriately: for example,
+      <literal>caas-admin</literal>, <literal>caas-master</literal>, 
+      <literal>caas-worker1</literal>, <literal>caas-worker2</literal> and so
+      on.
+     </para>
+     <sect5>
+      <title>Configuration Files</title>
+      <para>
+       There are two separate configuration files: <literal>user-data</literal>
+       and <literal>meta-data</literal>. Each node needs both. Thus, you must at
+       a minimum prepare one pair of files for the &admin_node;, and another
+       pair of files for all the &worker_node;s.
+      </para>
+      <para>
+       The two files must be placed into a subdirectory. For the &admin_node;,
+       this must be called <literal>cc-admin</literal>. For the &worker_node;s,
+       this must be called <literal>cc-worker</literal>.
+      </para>
+      <para>
+       So, for instance, if your working directory is called <literal>~/cloud-config</literal>,
+       for the admin node, you need:
+      </para>
+      <screen>~/cloud-config/cc-admin/user-data</screen>
+      <para>
+       and
+      </para>
+      <screen>~/cloud-config/cc-admin/meta-data</screen>
+      <para>
+       For a &worker_node;, you need:
+      </para>
+      <screen>~/cloud-config/cc-worker/user-data</screen>
+      <para>
+       and
+      </para>
+      <screen>~/cloud-config/cc-worker/meta-data</screen>
+      <para>
+       The same <literal>meta-data</literal> file can be used for both node types.
+       Here is an sample <literal>meta-data</literal> file:
+       </para>
+      <screen>
+# File: cloud-config/cc-worker/meta-data
+instance-id: <replaceable>iid-CAAS01</replaceable>
+network-interfaces: |
+  auto <replaceable>eth0</replaceable>
+  iface <replaceable>eth0</replaceable> inet dhcp
+       </screen>
+      <para>
+       The <literal>user-data</literal> file contains settings such as time
```

Comment:
use the <filename/> tag rather than <literal/> for filenames

Reply:
trying again... use the  `<filename/>`  tag for filenames

Reply:
A new tag on me! Thanks. Changed.